index.css

```diff
@@ -13,13 +13,17 @@ section {
 }
 
 /* footer section */
+footer {
+  margin-bottom: 20px;
+}
+
 .love {
```

Comment:
We should add a #footer tag before this in case the class name is used elsewhere